cpp/plugins/cucim.kit.cuslide/CMakeLists.txt

```diff
@@ -125,8 +125,6 @@ superbuild_depend(cli11)
 superbuild_depend(pugixml)
 superbuild_depend(json)
 superbuild_depend(libdeflate)
-superbuild_depend(nvjpeg)
-superbuild_depend(libculibos)
```

Comment:
These handled finding the nvJPEG and cuLIBOS dependencies via custom CMake logic

That is no longer needed now that they come from the CTK

Hence this and the associated files are dropped